src/sqswatcher/plugins/sge.py

```diff
@@ -99,5 +103,52 @@ def update_cluster(max_cluster_size, cluster_user, update_events, instance_prope
     return failed, succeeded
 
 
+def _is_node_locked(hostname):
+    try:
+        node_info = get_compute_nodes_info(hostname_filter=hostname)
+        node = node_info.get(socket.getfqdn(hostname), node_info.get(hostname))
+        if SGE_DISABLED_STATE in node.state:
```

Comment:
Using `in` instead of `==` is by design, because a node can have hybrid states and we just want to check if the locking state is in the hybrid states in order to check if a node is locked or not.